### ISSUE TITLE
support for responsive image preloading

### ISSUE THREAD

**Issue by danielroe** [nuxt/image#487]
https://web.dev/preload-responsive-images/

Example:

```html
<link rel="preload" as="image" href="wolf.jpg" imagesrcset="wolf_400px.jpg 400w, wolf_800px.jpg 800w, wolf_1600px.jpg 1600w" imagesizes="50vw">
```

**Comment by pi0** [comment]
This is different than `preload` prop? https://github.com/nuxt/image/pull/444

**Comment by andreavettore** [comment]
@danielroe please guys fix the 0.6.2 nuxt-image version. Image preloading is a huge feature.

**Comment by danielroe** [comment]
@andreavettore I'm not sure what you are asking. Preload support was merged in https://github.com/nuxt/image/pull/444 and released in 0.6.1. If you are experiencing an issue with it, please open one with a reproduction. 🙏
